### DISCUSSION TITLE
get public key as string

### DISCUSSION THREAD

**Discussion by majdslmt** [MasterKale/SimpleWebAuthn#551] [Troubleshooting]
How to get the public key as string  from AuthenticatorDevice ?

**Comment by MasterKale** [comment]
Hello @majdslmt, I'd recommend using the `isoBase64URL` helper in **@simplewebauthn/server**:

```ts
import { isoBase64URL } from '@simplewebauthn/server/helpers';

const base64urlPubKey = isoBase64URL.fromBuffer(authenticator.credentialPublicKey);
```

`base64urlPubKey` will be a string you can share around or store as needed.
